src/format/stream.rs

```diff
@@ -90,6 +92,11 @@ impl Stream {
         }
     }
 
+    /// Get stream id.
+    pub fn get_stream_id(&self) -> i32 {
```

Comment:
to be consistent with the other methods:
```suggestion
    pub fn stream_id(&self) -> i32 {
```

Reply:
Done, thanks :)